docs/design_decisions.md

```diff
@@ -148,26 +136,23 @@ _**Ratio**_
 Implementatie en gebruik eenvoudig houden. Er is geen functionele behoefte om diep gegevens te embedden.
 Het opvragen van relaties is eenvoudig. Bij dieper embedden kan doelbinding een probleem worden. Bij dieper embedden kunnen er aan de provider-kant performanceproblemen ontstaan. 
 
-### DD3.6 De identificatie van de gerelateerde resources worden opgenomen in de content van de opgevraagde resource
+### DD3.4 De identificatie van de gerelateerde resources worden opgenomen in de content van de opgevraagde resource
 Voor developers die geen HAL links willen gebruiken wordt tevens de identificatie van de gerelateerde resource opgenomen in de content van de opgevraagde resource.
 
 ## Historie
 
 ### DD4.1 Historie wordt gesorteerd op geldigheid met het meest actuele resultaat bovenaan
 Historie wordt aflopend gesorteerd op datum geldigheid (datumVan).
 
-### DD4.2 Historie elementen krijgen dezelfde naam en betekenis als in de NEN3610
-Historie elementen krijgen de namen _'beginGeldigheid'_ en _'eindGeldigheid'_.
-Binnen de NEN3610 standaard is er een keuze-mogelijkheid (datum en datumtijd) voor het formaat waarin de historie wordt bijgehouden. De Bron houdt geen tijd bij dus schrijven we het datum-formaat voor. 
-
-### DD4.3 Bij historie wordt alleen de actuele situatie van inOnderzoek getoond
+### DD4.2 Bij historie wordt alleen de actuele situatie van inOnderzoek getoond
 Binnen de historie-endpoints wordt alleen de actuele situatie met betrekking tot "in Onderzoek" getoond. Er wordt geen historie getoond van de onderzoeken die in het verleden hebben plaatsgevonden.  
 
 ## Diversen
 
 ### DD5.1 Descriptions worden als sibling van $Ref's opgenomen
 
-In OAS3 wordt er een warning gegeven als er binnen een object een Description wordt opgenomen naast een $ref (Geen enkele sibling is geoorloofd). In de draft van OAS3.1 is een description binnen een object naast een $ref wel geaccepteerd. Vooruitlopend op deze versie is het binnen de HaalCentraal-specificaties geaccepteerd dat er binnen een object naast een $ref een Description wordt opgenomen en wordt de warning voor lief genomen. [Zie hier](https://github.com/OAI/OpenAPI-Specification/issues/2033) voor een toelichting.
+Waar in OAS3 een Description binnen een object naast een $ref niet toegestaan (daar is geen enkele sibling geoorloofd) is wordt deze in de draft van OAS3.1 wel toegestaan. Vooruitlopend op deze nieuwe versie is het binnen de HaalCentraal-specificaties geaccepteerd dat er binnen een object naast een $ref een Description wordt opgenomen. 
+In OAS3 wordt er in deze gevallen een warning gegeven, aangezien dit soort descriptions echter binnen de geresolvde versie van het yaml bestand niet meer voorkomen wordt deze warning voor lief genomen. [Zie hier](https://github.com/OAI/OpenAPI-Specification/issues/2033) voor een toelichting.
```

Comment:
Om te zorgen dat de description toch wordt getoond (onder OAS3.0) wordt deze ook opgenomen in de description van het schema component waar de in is gedefinieerd.

Reply:
Heb ik toegevoegd.